docs/examples/301/modules-vwan-to-vnet-s2s-with-fw/main.bicep

```diff
@@ -0,0 +1,206 @@
+targetScope = 'subscription'
+
+param location string {
+  default: 'westeurope'

```

Comment:
is there a reason the two locations default to `westeurope` and `eastus` as opposed to `resourceGroup().location`?

Reply:
I'm not allowed to use resourceGroup().location as default value for the parameters when the targetScope is set to subscription. And I want the Virtual WAN hub and VNet to be in different Azure Regions. Maybe there's a way to do it that I'm missing.